Emby.Server.Implementations/Channels/RefreshChannelsScheduledTask.cs

```diff
@@ -18,27 +19,30 @@ public class RefreshChannelsScheduledTask : IScheduledTask, IConfigurableSchedul
         private readonly IUserManager _userManager;
         private readonly ILogger _logger;
         private readonly ILibraryManager _libraryManager;
+        private readonly ILocalizationManager _localization;
 
         public RefreshChannelsScheduledTask(
             IChannelManager channelManager,
             IUserManager userManager,
             ILogger<RefreshChannelsScheduledTask> logger,
-            ILibraryManager libraryManager)
+            ILibraryManager libraryManager,
+            ILocalizationManager localization)
         {
             _channelManager = channelManager;
             _userManager = userManager;
             _logger = logger;
             _libraryManager = libraryManager;
+            _localization = localization;
         }
 
         /// <inheritdoc />
-        public string Name => "Refresh Channels";
+        public string Name => _localization.GetLocalizedString("TasksRefreshChannels");
 
         /// <inheritdoc />
-        public string Description => "Refreshes internet channel information.";
+        public string Description => _localization.GetLocalizedString("TasksRefreshChannelsDescription");
 
         /// <inheritdoc />
-        public string Category => "Internet Channels";
+        public string Category => _localization.GetLocalizedString("TasksCategoryChannels");
```

Comment:
I don't exactly like the proposed naming scheme - in `Description` the word `Description` is postfixed to the task name, and in `Category` it's prefixed.
I propose to name them (for task `Foo`) like this:
 - `Name` => `TaskFooName`
 - `Description` => `TaskFooDescription`
 - `Category` => `TaskFooCategory`

Reply:
`TaskFooName` makes sense, but the categories aren't associated with a specific task, there are only four of them "Maintenance",  "Library", "Application", and "Internet Channels", and each one is associated with multiple tasks

Reply:
In this case name it `TaskBarCategory` for category `Bar`. My point is, this should always be a suffix, not a mish-mash of suffix vs prefix.

Reply:
@JustAMan Now I see your point. Changes being pushed right now...